src/login/logind-dbus.c

```diff
@@ -369,6 +369,33 @@ static int property_get_scheduled_shutdown(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_maintenance_window(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        _cleanup_free_ char *s = NULL;
+        Manager *m = userdata;
+        int r = 0;
+
+        assert(bus);
+        assert(reply);
+        assert(m);
+
+        if (m->maintenance_window)
+                r = calendar_spec_to_string(m->maintenance_window, &s);
+        if (r < 0) {
+                log_error_errno(r, "Failed to format calendar specification: %m");
+                return r;
```

Comment:
Please use `return log_error_errno(r, "...");`.

BTW, not wrong, but the error handling looks super spurious for me. Please do like the following:
```c
if (m->maintenance_window) {
        r = calendar_spec_to_string(...);
        if (r < 0)
                return log_error_errno(r, "...");
}

return sd_bus_message_append(...);
```